charts/rqlite/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v2
+name: rqlite
+version: 0.1.0
+appVersion: 8.13.2
```

Comment:
So the chart is pinned to a certain version of rqlite?

Reply:
By default, but the user can override the rqlite version with `image.tag`.  The chart will rev with rqlite periodically to ensure compatibility.

Reply:
That is to say, if the user overrides `image.tag` there's no guarantee of compatibility between the chart version and the rqlite version.  But of course most of the time it'll work fine, unless rqlite itself introduces a breaking change.

Reply:
Well, at least go to 8.13.4, where ServerName actually works (I think!) :-)

Reply:
Yeah I'll keep reving the chart as rqlite revs.  You're just much faster than I am. :)